packages/lodestar/test/unit/api/impl/config/config.test.ts

```diff
@@ -0,0 +1,34 @@
+import {config} from "@chainsafe/lodestar-config/minimal";
+import {expect} from "chai";
+import {ConfigApi} from "../../../../../src/api/impl/config";
+
+describe("config api implementation", function () {
```

Comment:
Add unit tests for rest api endpoints as well (there is plenty of examples already)